### ISSUE TITLE
Why there‘s no output?

### ISSUE THREAD

**Issue by Nimunru** [rabbitstack/fibratus#27]
I'm using Windows 10, Python34.
I think I have strictly followed the installation steps, but it always stops without result output and any error info, as the following picture:

![image](https://user-images.githubusercontent.com/22410372/63503307-cdce4f80-c501-11e9-8169-8795412e12dc.png)

By the way, I don't think the warning of pyyaml has impact on my problem, because using pyyaml==1.1.0 will solve this warning but still no output.

**Comment by rabbitstack** [comment]
I assume you got it compiled from sources by yourself? Is the issue reproducible if you install Fibratus from standalone installer?

**Comment by Nimunru** [comment]
> I assume you got it compiled from sources by yourself? Is the issue reproducible if you install Fibratus from standalone installer?
Still the same problem.  (╥_╥)
![image](https://user-images.githubusercontent.com/22410372/63505746-a928a680-c506-11e9-9985-6a0ebfa54f06.png)

**Comment by rabbitstack** [comment]
Does `.fibratus/fibratus.log` reveal anything useful? 

It is unlikely to happen, but maybe something is meddling with the `NT Kernel Logger` ETW session.
Could you please run `logman  query -ets` before and after starting Fibratus?

**Comment by Nimunru** [comment]
Sorry so late to reply.
This is the content of  `.fibratus\fibratus.log `:
![D~T~7_AAV@29LS@5C4K5((G](https://user-images.githubusercontent.com/22410372/63561703-7970b180-c58d-11e9-8e0a-8e8dacf54245.png)

This is `logman query -ets` before Fibratus:
![H CN@7H__5VFLJLCPW5{_NJ](https://user-images.githubusercontent.com/22410372/63562037-b7220a00-c58e-11e9-87de-07f8ea5f0309.jpg)

This is the after:
![@Y58%$DZVCU4%4Q2QOKQLQA](https://user-images.githubusercontent.com/22410372/63562039-ba1cfa80-c58e-11e9-8257-a89b0bd32180.jpg)

**Comment by rabbitstack** [comment]
Not sure what is going wrong. The `NT Kernel Logger` session is started successfully, yet Fibratus seems to terminate immediately. I'll try to reproduce on my machine and let you know the findings.

**Comment by rabbitstack** [comment]
Fibratus apparently crashes in Cython extension that is responsible for consuming kernel events:

```
Faulting application name: fibratus.exe, version: 0.0.0.0, time stamp: 0x00021856
Faulting module name: kstreamc.pyd, version: 0.0.0.0, time stamp: 0x5973a6ad
Exception code: 0xc0000005
Fault offset: 0x00000000000038a2
```
I'm not sure if I'll have the luxury to invest some time into this as I'm migrating Fibratus to Go as announced in README.

**Comment by Nimunru** [comment]
Anyway，thanks for your reply. Hopes it will be solved.

**Comment by rabbitstack** [comment]
I'll try to deliver the fix in the upcoming weeks. This needs to be something related to recent Windows 10 builds since I never had issues running Fibratus in Windows 10 in the past year.

**Comment by Nimunru** [comment]
My windows 10 version is professional edition, 10.0.18362.295. I'm not sure whether the issue is caused by the system version or my system environment. 
Actually I have succeed in running Fibratus in Windows 7.
Now I have a new question that does Fibratus have the function of capturing user's activity, for example, manually creating a new file on the desktop?

**Comment by rabbitstack** [comment]
It is likely caused by Windows version. Some nasty discrepancy between Windows 10 Home/Pro versions in terms of ETW internals.

With Fibratus you can capture any of kernel events listed [here](https://github.com/rabbitstack/fibratus/wiki/Filtering). To watch on created files, you could use an existing [filament](https://github.com/rabbitstack/fibratus/blob/master/filaments/created_files.py):

`fibratus run --filament=created_files`

**Comment by rabbitstack** [comment]
@Nimunru this should be fixed in [1.0.0](https://github.com/rabbitstack/fibratus/releases/tag/1.0.0).
